src/components/Header/Header.tsx

```diff
@@ -0,0 +1,45 @@
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  query: string;
+  handleChange: (event: React.ChangeEvent<EventTarget>) => void;
+  onSumbit: (event: React.FormEvent<HTMLFormElement>) => void;
+  IsDisabled: boolean;
+  toggleAll: () => void;
+  isTodoStatus: boolean;
+};
+
+export const Header: React.FC<Props> = React.memo(({
+  query,
+  handleChange,
+  onSumbit,
+  IsDisabled,
+  toggleAll,
+  isTodoStatus,
+}) => {
+  return (
+    <header className="todoapp__header">
+      <button
+        type="button"
+        className={classNames('todoapp__toggle-all',
+          { active: !isTodoStatus })}
+        aria-label="toggle-all"
+        onClick={() => toggleAll()}
+      />
+
+      <form
+        onSubmit={(event) => onSumbit(event)}
```

Comment:
the same here: `onSubmit={onSubmit}`.
It's not an error but only a syntax improvement.